### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by IBUNHABIBU**
This app user is able to do the following
- Create, select and delete the projects.
- Display list of project
- Each project has a list of task.
- User is able to create and delete task as well as to mark the completed tasks.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @IBUNHABIBU 
### You did well so far :clap: However, you have a few issues to fix:-
- [ ] Create a default project, use local storage to achieve this as it is a requirement as well
- [ ] I am unable to create a todo item as it does not save, the date also is not working, check the issue, and kindly fix it. Refer the image below:
todo
- [ ] Update your scripts in the package.json, refer the code below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
- [ ] Use es6 arrow functions. see below
```
const renderTasks = (selectedProject) => {
   # your codebase
}
```
- [ ] Separate app logic from dom manipulation
- [ ] All the required linting dependencies are not installed, therefore, it returns some errors, [follow this guide](https://github.com/microverseinc/linters-config/tree/master/javascript)
### Make the changes and submit the project for another code review. Good luck.
- Happy coding!

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+}
+
+#datetime {
+  width: 9.6rem;
+}
+
+/* input[type='date']::before {
+  content: attr(placeholder);
+} */
+
```
Comment (MuriloRoque) [inline src/css/style.css]: - [ ] Please remove commented lines of code.

File: src/modules/app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import '../css/style.css';
+
+import Project from './project';
+
+function Tasker() {
```
Comment (MuriloRoque) [inline src/modules/app.js]: - [ ] Please use ES6 syntax.

File: src/modules/clear.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const Clear = () => {
+  function clearPrevious(projectList) {
```
Comment (MuriloRoque) [inline src/modules/clear.js]: - [ ] Please use ES6 syntax.

File: src/modules/localDb.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+function LocalSaver() {
```
Comment (MuriloRoque) [inline src/modules/localDb.js]: - [ ] Please use ES6 syntax.

File: src/modules/project.js [hunk 5]
```diff
@@ -0,0 +1,165 @@
+import Clear from './clear';
+
+import LocalSaver from './localDb';
+
+function Project() {
```
Comment (MuriloRoque) [inline src/modules/project.js]: - [ ] Please use ES6 syntax.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @IBUNHABIBU :wave: 

### You are making progress, I see that you made some of the previously requested changes, but we still need to fix somethings:

- [ ] I can't create a todo item, it gives a console error if I try clicking the button:

Screenshot from 2020-10-12 13-39-01

- [ ] If I delete a project when I refresh my browser it will still be there, is it really deleting from the local storage?

- [ ] Separate your logic from the DOM manipulation code, put them in separate files. Every time you select a DOM node, like `document.getElementById('id')`, you are playing with the DOM, you should create another file only for those lines of code. I know this is one of the hardest things to achieve, especially when you already started your project, but it's a good practice and not impossible :muscle: 

- [ ] Please check my comments in your code below.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by IBUNHABIBU** [comment]
-User can delete and mark completed task though sometimes you need to refresh the browser.
-I used the arrow function.
-I have put the Dom in a separate file.
Thanks in advance I would appreciate your feedback and help

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @IBUNHABIBU 👋

**Nice work so far, fixing a few of the changes requested.**

-  [ ] It's not possible to delete a to-do item on a project, please fix that.

- [ ] Also clicking on high priority doesn't create a to-do with high priority. Kindly correct this.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/css/style.css (hunk 1 above)
Comment (IBUNHABIBU) [reply src/css/style.css]: -I have made the delete task to work properly
-I have made the priority to work both high and low by choosing the radio button

**Comment by Dandush03** [comment]
# Status: :rocket: Project Approved :rocket: 

### Great Work! Keep It Up :muscle: 

# Happy Coding! :nerd_face: :computer:

**Pull request merged**